components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
```

Comment:
> From the previous discussion my impression was that we want to remove usage of build_absolute_uri

I agree, I was just a bit hesitant to do the full cleanup because I don't understand why the `else` is there. I'll remove it if no-one can objects.